src/common/input-map/input-map.scss

```diff
@@ -12,8 +12,6 @@
     padding-left: 0px;
     >*{
         display: inline-block;
-    }
-    label, input{
-        width: $sidebar-width;
+        width: 48.2%;
```

Comment:
"48.2%" is a pretty magic number. Could we avoid this and use flexbox to control the size and layout?